optuna/integration/chainermn.py

```diff
@@ -124,6 +124,11 @@ def optimize(
         the absence of ``n_jobs`` argument.
         """
 
+        if n_trials:
+            if n_trials > 1000:
+                warnings.warn("visualization features might not be stable "
+                              "n_trials is more than 1000")
```

Comment:
Is the number always around 1,000?

In case it's not true, how about just mentioning, for example: `Plotting {} trials, which may cause unstable behavior of the visualization feature with too many trials.`